.github/workflows/tests.yml

```diff
@@ -94,6 +94,19 @@ jobs:
           target="not slow"
         fi
 
-        pytest tests -m "$target"
+        if [ "${{ matrix.python-version }}" = "3.13" ] ; then
+          pytest tests -m "$target" \
+            --ignore=tests/gp_tests/ \
+            --ignore=tests/samplers_tests/test_gp.py \
+            --ignore=tests/terminator_tests/improvement_tests/test_evaluator.py \
+            --ignore=tests/terminator_tests/improvement_tests/test_evaluator.py \
+            --ignore=tests/samplers_tests/test_samplers.py \
```

Comment:
The `test_samplers.py` includes a lot of tests for other samplers. How about installing the nightly version of `torch` in the CI, and do not skip all tests?